Topics/Tech_Stacks/Apache_Superset.md

```diff
@@ -38,6 +42,46 @@ This message shows up due to an update in version 2.1.0 to force secure configur
 1. The command `pip install apache-superset` doesn't work. This is because Apache Superset currently supports python version 3.8 and 3.9. Any python versions that's lower or higher will result in a failure.
 
 
+## Creating a Custom Plugin:
```

Comment:
Are there any specific requirements in order to create a custom plug-in?

Reply:
To set up for the plug-in, you would need to have the following in your system: 
apache-superset 3.0.0
python 3.9.7 or above
node version 16
npm version 7 or 8

I will add these requirements in the md file. Thanks for your comment.